validator/src/main/resources/expected-data-template/java/ec2/asg/aws-sdk-call-log.mustache

```diff
@@ -0,0 +1,19 @@
+[{
+  "EC2.AutoScalingGroupName":  "^{{platformInfo}}$",
+  "HostedIn.EC2.Environment": "^{{platformInfo}}$",
+  "Operation": "GET /aws-sdk-call",
```

Comment:
Check for `EC2.InstanceId` as well in all logs

Reply:
`host.name` should also appear, we need to validate it as well

Reply:
Done